db/migrations/20180627220827_create_price_for_line_items.cr

```diff
@@ -0,0 +1,22 @@
+class CreatePriceForLineItems::V20180627220827 < LuckyMigrator::Migration::V1
+  def migrate
+    # Needs support in lucky_migrator
+    # create :prices, primary_key_type: :uuid do
+    #   add in_cents : Int32
+    #   add line_item_id : UUID
```

Comment:
For the comment here, could you create an issue on LuckyMigrator?

Also, I think it would be something like this:

```
add_belongs_to line_item : LineItem, foreign_key_type: UUID
```

So it can set up foreign keys and all that jazz as well

Reply:
Done, I just found that out as well. So I guess we can either leave this comment and merge, or wait until lucky_migrator has support ...

Reply:
I think removing the comment and adding support for it in LuckyMigrator later would be ok.

That way we can at least make it available for people now, and refine the experience later. We may even have time to add it in LuckyMigrator before release :)

Reply:
Sure, done